components/automate-ui/src/app/pages/infra-proxy/chef-servers-details/chef-servers-details.component.scss

```diff
@@ -0,0 +1,129 @@
+@import "~styles/variables";
+
+chef-page-header {
+  padding-bottom: 0;
+}
+
+chef-toolbar {
+  display: block;
+  position: relative;
+  margin-bottom: 5px;
+
+  small {
+    bottom: 2px;
+    color: $chef-dark-grey;
+    position: absolute;
+    right: 0;
+  }
+}
+
+chef-tab-selector {
+  chef-option {
+    margin-right: 16px;
+    color: $chef-primary-dark;
+  }
+}
+
+form {
+
+  #update-name {
+    width: 240px;
+  }
+
+  input {
+    font-size: 1em;
```

Comment:
FYI for in the near future, `font-size` on `chefInput` directives _shouldn't_ be needed, and will be handled by the chefInput directive at 14px. 😄